### ISSUE TITLE
Integrate accusation manager with the new Task scheduler

### ISSUE THREAD

**Issue by ghost** [alicenet/alicenet#383]
One of the final responsibilities of the accusation manager is to accuse malicious validators by sending accusations to our solidity smart contracts. To do so the accusation manager needs to spawn a Task that will carry this last step in the process. At the moment, the Task management is undergoing a refactoring and the accusation manager is not yet capable of finishing the accusation process. Once the new Task manager is in place we still need to integrate with it.

Blocked by PR: [Refactor the layer1 package by vtleonardo · Pull Request #11 · alicenet/alicenet](https://github.com/alicenet/alicenet/pull/11) 

**Acceptance criteria:**

- The accusation manager can submit accusation tasks to the Task Scheduler
- Accusation tasks sent to the Task Manager should implement a Task interface
- The Task Manager should receive a Task interface object to keep it agnostic
- No concrete accusation Task is needed for the purposes of this story, i.e., in the future each accusation story should also entail the building of such task
- Once an accusation task is submitted to the Task Manager, there needs to be some sort of confirmation from the Task Manager indicating the task is persisted and resilient to node restarts/crashes. Depending on how it works it can be a blocking function call or an asynchronous channel based confirmation
- After an accusation task is submitted to the Task Manager and such confirmation is received by the Accusation Manager, the Accusation Manager should not retry sending this task to the Task Manager
- Sending a duplicate accusation task to the Task Manager should not be possible. The accusation UUID should be the unique identifier. This is to avoid double accusations in the event of a node crash/restart in the middle of this communication
